### DISCUSSION TITLE
Compatibility Inquiry: Can Open DTU and Hoymiles DTU Operate Simultaneously?

### DISCUSSION THREAD

**Discussion by NoCodeMaster** [tbnobody/OpenDTU#1100] [Q&A]
I'm thinking of installing Open DTU in my system, but since the Hoymiles DTU is still under warranty and the installation company is still extracting data from it to verify that everything is functioning correctly, I would like to know if both systems can work in parallel, meaning if they can read data from both Open DTU and the Hoymiles DTU.

Can anyone help me with this?

Thank you in advance.

**Comment by Solarteur** [comment]
Operating an OpenDTU with an additional DTU (e.g. Hoymiles) incorrect values are displayed. See also https://github.com/tbnobody/OpenDTU/wiki/FAQ#q-opendtu-shows-wrongunrealistic-or-just-missing-values

and 

https://github.com/tbnobody/OpenDTU#troubleshooting
(last paragraph).
